subsys/zbus/zbus.c

```diff
@@ -12,6 +12,16 @@
 #include <zephyr/zbus/zbus.h>
 LOG_MODULE_REGISTER(zbus, CONFIG_ZBUS_LOG_LEVEL);
 
+#define _THREAD_NAME                                                                               \
+	(IS_ENABLED(CONFIG_THREAD_NAME) ? k_thread_name_get(k_current_get()) : "publisher")
```

Comment:
```c
#if CONFIG_THREAD_NAME
#define ZBUS_THREAD_NAME k_thread_name_get(k_current_get())
#else
#define ZBUS_THREAD_NAME "publisher"
#endif
```

I think _ prefix deifnes should be avoided, `IS_ENABLED` is a bit of an odd thing, if normal ifdef works just use those

Reply:
Removed.